### ISSUE TITLE
in touch screen pc chrome , the step can not  be dragged

### ISSUE THREAD

**Issue by guojingkang** [jsplumb/jsplumb#919]
the step will cant not be dragged on some touch pc （window&chrome）
i find the problem will be because of  the judgement of the touch device in the package `mottle`
```code 
isTouchDevice = "ontouchstart" in document.documentElement
```
i change it to 
```
isTouchDevice = "ontouchstart" in document.documentElement || navigator.maxTouchPoints
```
and the problem is fixed.

**Comment by sporritt** [comment]
that's very useful, thanks - I'll release a new version today with this included.

**Comment by sporritt** [comment]
fixed and published in jsplumb 2.13.0
